libs/core/src/test/java/org/elasticsearch/common/unit/TimeValueTests.java

```diff
@@ -165,7 +166,7 @@ public void testNoDotsAllowed() {
     }
 
     public void testToStringRep() {
-        assertEquals("-1", new TimeValue(-1).getStringRep());
+        assertEquals("-1ms", new TimeValue(-1).getStringRep());
```

Comment:
I think we want to keep this one rendered as `-1`. Otherwise we're saying that `-1ms` is the canonical representation of a missing value, but that's not true, we want to discourage the use of minus-one-with-units as much as possible.